### ISSUE TITLE
RTL Language Bullets Display Error

### ISSUE THREAD

**Issue by umertanvir** [mozilla-mobile/focus-android#788]
Focus is displaying bullets on left for Urdu Language. As Urdu is RTL Language so bullets should be placed at right side 
![unnamed](https://user-images.githubusercontent.com/13235124/27219438-26a65b9c-529b-11e7-85ec-a223e61f4d2c.jpg)

**Comment by liuche** [comment]
Took a look at the css, we shouldn't be using margin-left/right, but rather start-end.

**Comment by umertanvir** [comment]
Hopefully it get fixed

**Comment by pocmo** [comment]
This is not a HTML / CSS problem (-> It works "most of the time").

@umertanvir Is your Android system language "Urdu" or did you use the language switcher in Focus (Settings)?

**Comment by umertanvir** [comment]
Hi My mobile default language is English. I used language switch within application

**Comment by pocmo** [comment]
@umertanvir Thanks! Does this problem resolve itself after opening this screen multiple times or after going to settings (again) before opening the about screen?

What device / Android version are you using?

**Comment by umertanvir** [comment]
No i have to goto settings and **About** section and the problem is not resolved at all. I am using xiaomi redmi 3s / Android version 6.0.1

**Comment by umertanvir** [comment]
Problem still  remain  in Both Focus Beta v1.1 and Focus

**Comment by ekager** [comment]
I experimented with this today by switching the Urdu string for the working Arabic string, and the bullet side switches to RTL. This makes me believe that this problem is something about the Urdu (and Persian) string which is formatted as CDATA, and not about the Urdu or Persian language detection. I'm not sure where l10n creates these strings but I think there is a missing tag of some sort for Urdu and Persian. I was able to force the RTL bullets and formatting by adding a `<div dir="rtl">` tag, but we should verify that the translation stayed the same with l10n.
